format/spec.md

```diff
@@ -454,35 +457,40 @@ The schema of a manifest file is a struct called `manifest_entry` with the follo
-| _optional_ | _optional_ | **`109  value_counts`**           | `map<119: int, 120: long>`   | Map from column id to number of values in the column (including null and NaN values) |
-| _optional_ | _optional_ | **`110  null_value_counts`**      | `map<121: int, 122: long>`   | Map from column id to number of null values in the column |
-| _optional_ | _optional_ | **`137  nan_value_counts`**       | `map<138: int, 139: long>`   | Map from column id to number of NaN values in the column |
-| _optional_ | _optional_ | **`111  distinct_counts`**        | `map<123: int, 124: long>`   | Map from column id to number of distinct values in the column; distinct counts must be derived using values in the file by counting or using sketches, but not using methods like merging existing distinct counts |
-| _optional_ | _optional_ | **`125  lower_bounds`**           | `map<126: int, 127: binary>` | Map from column id to lower bound in the column serialized as binary [1]. Each value must be less than or equal to all non-null, non-NaN values in the column for the file [2] |
-| _optional_ | _optional_ | **`128  upper_bounds`**           | `map<129: int, 130: binary>` | Map from column id to upper bound in the column serialized as binary [1]. Each value must be greater than or equal to all non-null, non-Nan values in the column for the file [2] |
-| _optional_ | _optional_ | **`131  key_metadata`**           | `binary`                     | Implementation-specific key metadata for encryption |
-| _optional_ | _optional_ | **`132  split_offsets`**          | `list<133: long>`            | Split offsets for the data file. For example, all row group offsets in a Parquet file. Must be sorted ascending |
-|            | _optional_ | **`135  equality_ids`**           | `list<136: int>`             | Field ids used to determine row equality in equality delete files. Required when `content=2` and should be null otherwise. Fields with ids listed in this column must be present in the delete file |
-| _optional_ | _optional_ | **`140  sort_order_id`**          | `int`                        | ID representing sort order for this file [3]. |
+| v1         | v2         | v3         | Field id, name                    | Type                         | Description |
+| ---------- | ---------- | ---------- |-----------------------------------|------------------------------|-------------|
+|            | _required_ | _required_ | **`134  content`**                | `int` with meaning: `0: DATA`, `1: POSITION DELETES`, `2: EQUALITY DELETES` | Type of content stored by the data file: data, equality deletes, or position deletes (all v1 files are data files) |
+| _required_ | _required_ | _required_ | **`100  file_path`**              | `string`                     | Full URI for the file with FS scheme |
+| _required_ | _required_ | _required_ | **`101  file_format`**            | `string`                     | String file format name, `avro`, `orc`, `parquet`, or `puffin` |
```

Comment:
nit: we should note that puffin is only used for storing deletion vectors (this might not be obvious for first time readers, since all deletion vectors are only discussed below this point)

Reply:
Another question here, if the intent is really to be compatible with delta lake, maybe this should just be called "blob" storage isn't tied to puffin, and users have the ability to store this in other locations?

Reply:
After thinking about it some more, I think I'd be more in favor of keeping content [type above](https://github.com/apache/iceberg/pull/11240/files#r1796418244) and calling this `delete-vector-v1-blob`.  Implementations still must write this as a puffin file but this leaves flexibility for readers without violating the spec.

Reply:
I replied to this in the thread above.

I think this field should continue to be the overall file format (puffin here) and that if anything we want a type for the blob. However, we only have one case right now so let's add that when we need it.

Reply:
OK, I think we should add a not that puffin is only used for storing DVs (maybe a link to the DV section?)